### ISSUE TITLE
New version 3.2 prevents "Show total number of messages" 

### ISSUE THREAD

**Issue by HansAxel** [protz/Manually-Sort-Folders#206]
After restarting Thunderbird 115 the new version 3.2 prevents the "Show total number of messages" display behind the folders.
Can this be corrected?
Sorry for my bad English.
HansAxel

Win11 Pro 22H2/TB 115.3.3 (64-Bit) portable

**Comment by grac-zh** [comment]
Yes, unfortunately exactly this bug is still present - see my thread: [https://github.com/protz/Manually-Sort-Folders/issues/202](url)

this addon for thunderbird is not for use ...

**Comment by nonilevinson** [comment]
Excuse my dumb question, but where can I find the version 3.2 to download?
I am still on 2.3 (not working) on TB 115.4.2
Tks

**Comment by HansAxel** [comment]
I found it here.
https://github.com/protz/Manually-Sort-Folders/issues/199#issuecomment-1641133891
Thanks to @jobisoft

**Comment by nonilevinson** [comment]
@HansAxel 
Tks!!!

**Comment by TJR032354** [comment]
Understand before you spend a lot of time with 3.2 it won’t be totally
compatible with TB115. It was a valiant effort but not ready for prime
time. For example, it won’t allow you to sort sub-folders.

Just giving you a heads up.

**Comment by nonilevinson** [comment]
@TJR032354 
Tks for the advice!
Do you know any plans for a stable version to be released?

**Comment by TJR032354** [comment]
Unfortunately I don’t know when we’ll see a finished product. There was
some talk about modifying TB itself and eliminate the need for an add-on. I
have no clue if/when that might be done.

**Comment by nonilevinson** [comment]
ah! ok
Tks a lot!

**Comment by schneeschmelze** [comment]
That's right. MSF will become part of Thunderbird core, I understand: https://thunderbird.topicbox.com/groups/developers/T47d1b6f7d4f5047c-M1ae3ee7daca65d207ee17be1

**Comment by schneeschmelze** [comment]
> Target Milestone: --- → 121 Branch

https://bugzilla.mozilla.org/show_bug.cgi?id=1846550

**Comment by itagagaki** [comment]
> > Target Milestone: --- → 121 Branch
> 
> https://bugzilla.mozilla.org/show_bug.cgi?id=1846550

It was backed out due to performance issues with huge numbers of folders. I plan to bring it back as soon as the issue is fixed, but the timing is delicate that it may or may not go into the 121 beta. It may be the 122.
